interface/usergroup/user_admin.php

```diff
@@ -462,6 +462,9 @@ function authorized_clicked() {
   <?php echo generate_select_list("erxrole", "newcrop_erx_role", $iter['newcrop_user_role'],'',xl('Select Role'),'','','',array('style'=>'width:150px')); ?>
 </td>
 </tr>
+<tr>
+<td><span class="text"><?php xl('Weno Provider ID','e'); ?>: </span></td><td><input type="text" name="erxprid" style="width:150px;"  value="<?php echo $iter["weno_prov_id"]?>"></td>
```

Comment:
Since new line, need to follow current security model for html escaping
``````
<td><span class="text"><?php echo xlt('Weno Provider ID'); ?>: </span></td><td><input type="text" name="erxprid" style="width:150px;"  value="<?php echo attr($iter["weno_prov_id"]); ?>"></td
``````